src/CBot/CBotVar/CBotVar.cpp

```diff
@@ -378,7 +378,7 @@ void CBotVar::SetInit(CBotVar::InitType initType)
 }
 
 ////////////////////////////////////////////////////////////////////////////////
-std::string CBotVar::GetName()
+const std::string& CBotVar::GetName()
 {
     return    m_token->GetString();
```

Comment:
This change could cause dangling reference bugs in the future. It's currently fine as `m_token` is never reassigned to, but you have to read the rest of the code to know that. Perhaps declare `CBotVar::m_token` as `CBotToken* const` instead of `CBotToken*`?